### PR TITLE
Reformat linear_algebra/Outer

### LINKED ISSUE #12369: Outer

### PR DESCRIPTION
**Pull request by asad0991**
resolves #12369

### CODE REVIEW

File: ivy/container/linear_algebra.py [hunk 1]
```diff
@@ -1763,6 +1763,57 @@ def static_outer(
...
+            ret
+                an ivy container whose shape is (..., N1, N2). The first x1.ndim-1 dimensions
+                have the same size as those of the input array x1 and the first x2.ndim-1
+                dimensions have the same size as those of the input array x2.
+
+            Example
+            -------
+            >>> x1 = ivy.array([[1, 2, 3], [4, 5, 6]])
+            >>> x2 = ivy.array([1, 2, 3])
+            >>> y = ivy.static_outer(x1, x2)
```
Comment (sherry30) [inline ivy/container/linear_algebra.py:1808]: Can you use container inputs for this example.
and you should call the function by doing `ivy.Container.static_outer(x1, x2)`

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,

Just a small change requested, the rest looks good

Thanks

File: ivy/container/linear_algebra.py (hunk 1 above)
Comment (asad0991) [reply ivy/container/linear_algebra.py]: Done Kindly Review Again

**Review by sherry30 — APPROVED** [review]
lgtm 👍 

Thanks for the contribution

**Pull request merged**